doozerlib/distgit.py

```diff
@@ -2453,8 +2453,8 @@ def _merge_source(self):
             owners = list(self.config.owners)
 
         # If upstream has not identified the BZ component, have the pipeline send them a nag note.
-        maintainer = self.metadata.get_maintainer_info()
-        if owners and not maintainer.get('component', None):
+        jira_project, jira_component = self.metadata.get_jira_info()
+        if jira_project == 'OCPBUGS' and jira_component == 'Unknown':
```

Comment:
do we still need to check owners? 
the component can also be set "none"

Reply:
> do we still need to check owners?

Good point. The job checking for `bz_notify` also checks `owners` before notifying the owners.
I think the upside of checking owners here is a defensive measure, but the downside is we lose the ability to manually analyze with `grep`.

> the component can also be set "none"

Could you give me an exact example?